pkg/minikube/node/start.go

```diff
@@ -146,7 +145,7 @@ func Start(starter Starter, apiServer bool) (*kubeconfig.Settings, error) {
 		}
 
 		// Skip pre-existing, because we already waited for health
```

Comment:
Remove this comment and add a TODO: "Pre-existing clusters should wait for health: #7597"